conda/environments/nvtabular_aws_sagemaker.yml

```diff
@@ -0,0 +1,22 @@
+# Based on https://github.com/NVIDIA-Merlin/NVTabular/blob/main/conda/environments/nvtabular_dev_cuda11.0.yml
+name: nvtabular
+channels:
+  - rapidsai
+  - nvidia
+  - conda-forge
+  - defaults
+dependencies:
+  - nvtabular
+  - python>=3.7
+  - cudatoolkit=11.0
```

Comment:
why cudatoolkit 11.0? shall we not go with 11.4 or 11.5 instead?

Reply:
It may works with 11.4 or 11.5. I adopted the original script from Eugene. AWS SageMaker has only a few predefined images and not sure, which CUDA version, they use